src/OrbitGl/App.cpp

```diff
@@ -2087,7 +2087,6 @@ void OrbitApp::RefreshFrameTracks() {
     GetMutableTimeGraph()->RemoveFrameTrack(function_id);
     AddFrameTrackTimers(function_id);
   }
-  GetMutableTimeGraph()->GetTrackManager()->RequestTrackSorting();
```

Comment:
Is this in here by accident? This will break the change Pablo & I did yesterday.

Reply:
yes, sorry